### ISSUE TITLE
Cannot find ckpt from None

### ISSUE THREAD

**Issue by galuhputraa** [peteryuX/arcface-tf2#54]
im have this error when running testing python test.py --cfg_path="./configs/arc_res50.yaml" , im already put the pretrained model checkpoint  on checkpoints folder and i just read closed issue but i still cant understand it , can u guys explain to me how to solve this problem?

**Comment by xalbertoisorna** [comment]
Yes, you have to specify a sub_folder in the yaml file that either contains a .cpkt or an empty one to start from 0

**Comment by xalbertoisorna** [comment]
> im have this error when running testing python test.py --cfg_path="./configs/arc_res50.yaml" , im already put the pretrained model checkpoint on checkpoints folder and i just read closed issue but i still cant understand it , can u guys explain to me how to solve this problem?

you have to specify a subfolder name "sub_name" in the yaml file that does contain a cpkt or an empty folder if you want to start from 0. In both cases the folder has to exist inside the checkpoint path
